src/applications/claims-status/utils/helpers.js

```diff
@@ -1064,9 +1064,13 @@ export const buildDateFormatter = (formatString = DATE_FORMATS.LONG_DATE) => {
   };
 };
 
+export const isAutomated5103Notice = itemDisplayName => {
```

Comment:
It feels like this function would be better if it took the entire `item` instead of just the displayName. That way, if there ends up being a better way to detect this in the future using other elements of the `item` you wouldn't need to refactor.

Something like `item => item.displayName === 'Automated 5103 Notice Response'`

Reply:
Initially I had it taking the whole `item` and not the `displayName` but changed that since IMO i didnt see us changing how we detect this via the item object and seemed like it wasn't necessary to pass in the whole item object every time.